tests/smoke-tests/src/test/java/org/apache/activemq/artemis/tests/smoke/brokerConnection/MirrorInfiniteRetryReplicaTest.java

```diff
@@ -153,18 +153,18 @@ private static void createMirroredServer(String serverName,
 
    private static void replaceLogs(File serverLocation) throws Exception {
       File log4j = new File(serverLocation, "/etc/log4j2.properties");
-      assertTrue(FileUtil.findReplace(log4j, "logger.artemis_utils.level=INFO",
-                                      "logger.artemis_utils.level=INFO\n" + "\n" +
-                                      "logger.endpoint.name=org.apache.activemq.artemis.core.replication.ReplicationEndpoint\n" +
-                                      "logger.endpoint.level=DEBUG\n" +
-                                      "logger.ackmanager.name=org.apache.activemq.artemis.protocol.amqp.connect.mirror.AckManager\n" +
-                                      "logger.ackmanager.level=INFO\n" +
-
-                                      "logger.mirrorTarget.name=org.apache.activemq.artemis.protocol.amqp.connect.mirror.AMQPMirrorControllerTarget\n" +
-                                      "logger.mirrorTarget.level=INFO\n" +
-
-                                      "appender.console.filter.threshold.type = ThresholdFilter\n" +
-                                      "appender.console.filter.threshold.level = trace"));
+      assertTrue(FileUtil.findReplace(log4j, "logger.artemis_utils.level=INFO", """
+         logger.artemis_utils.level=INFO
+
+         logger.endpoint.name=org.apache.activemq.artemis.core.replication.ReplicationEndpoint
+         logger.endpoint.level=DEBUG
+         logger.ackmanager.name=org.apache.activemq.artemis.protocol.amqp.connect.mirror.AckManager
+         logger.ackmanager.level=INFO
+         logger.mirrorTarget.name=org.apache.activemq.artemis.protocol.amqp.connect.mirror.AMQPMirrorControllerTarget
+         logger.mirrorTarget.level=INFO
+         appender.console.filter.threshold.type = ThresholdFilter
+         appender.console.filter.threshold.level = trace\
+         """));
```

Comment:
Possibly simpler to add the close deliminator on the same line than use line ending suppression?

Reply:
Fixed.